tests/test_upload.py

```diff
@@ -596,6 +602,69 @@ def test_marks_administrative_transactions(self, mock_get_conn):
         self.assertEqual(transactions[2]['source'], 'administrative')
         self.assertEqual(transactions[2]['batch'], 10)
 
+    @mock.patch('mtp_transaction_uploader.upload.settings')
+    def test_marking_all_credit_transactions_as_unidentified(self, mock_settings):
+        mock_settings.NOMS_AGENCY_ACCOUNT_NUMBER = '67175315'
+        mock_settings.NOMS_AGENCY_SORT_CODE = '123456'
+        mock_settings.MARK_TRANSACTIONS_AS_UNIDENTIFIED = True
```

Comment:
Might be worth adding this mock and explicitly setting this value to false in the other tests, otherwise they'll fail upon adding the environment variable that sets this to true